core/assets/bundles/bundle.properties

```diff
@@ -1175,6 +1176,7 @@ block.surge-tower.description = An extremely long-range power node with fewer av
 block.diode.description = Battery power can flow through this block in only one direction, but only if the other side has less power stored.
 block.battery.description = Stores power as a buffer in times of surplus energy. Outputs power in times of deficit.
 block.battery-large.description = Stores much more power than a regular battery.
+block.surge-capacitor.description = A massive and expensive battery, the surge capacitor uses surge plating for extremely high power storage.
```

Comment:
For the French translation: `block.surge-capacitor.description = Une batterie massive et coûteuse, le condensateur de surtension utilise un placage de sutension pour un stockage d'énergie extrêmement élevé.`.

Reply:
thank you